pkg/render/node.go

```diff
@@ -490,6 +536,99 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 	return role
 }
 
+// nodeRole creates the clusterrole containing policy rules that allow the node daemonset to operate normally.
+func (c *nodeComponent) CNIPluginRole() *rbacv1.ClusterRole {
```

Comment:
Comment for this function also looks like copy/paste


But this should also be called `cniPluginClusterRole`